contracts/apps/AragonApp.sol

```diff
@@ -1,4 +1,4 @@
-pragma solidity ^0.4.18;
+pragma solidity 0.4.18;
```

Comment:
The `AragonApp` contract is not pinned given that it might be imported by developers running a future (non-breaking) compiler version.